multicluster/build/yamls/new_sa.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: antrea-multicluster-member-access-sa
+  namespace: antrea-mcs-ns
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: antrea-multicluster-member-cluster-rolebinding
+  namespace: antrea-mcs-ns
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: antrea-multicluster-member-cluster-role
+subjects:
+- kind: ServiceAccount
+  name: antrea-multicluster-member-access-sa
+  namespace: antrea-mcs-ns
```

Comment:
missing new line, and `new_sa.yml` is not a proper name, you may just name it as `antrea-multicluster-member-access-sa.yml`